com.unity.ml-agents/Runtime/Actuators/ContinuousVectorActuator.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using Unity.MLAgents.Policies;
+
+namespace Unity.MLAgents.Actuators
+{
+    public class ContinuousVectorActuator : BaseActuator
+    {
+
+
+        public ContinuousVectorActuator(int[] vectorActionSize) : base("ContinuousVectorActuator",
```

Comment:
These classes just determine the number of actions from the vectorActionSize array from `BehaviorParameters`